WcaOnRails/db/structure.sql

```diff
@@ -1,8 +1,8 @@
--- MySQL dump 10.16  Distrib 10.1.9-MariaDB, for Linux (x86_64)
+-- MySQL dump 10.13  Distrib 5.5.46, for debian-linux-gnu (x86_64)
```

Comment:
`git reset --soft HEAD~1` gets you in the state right _before_ the commit (all changes are staged and `git status` tells you how to remove files from being staged).

For the future you can use `git add -p` before commiting. This lets you step through every single change and asks you whether you want to stage it or not. Great feature.